orchestra/static/dist/main.css

```diff
@@ -1355,3 +1369,183 @@ li.checklist-item, .new-item {
 .website-iframe {
   width: 100%;
   border: 1px solid #cccccc; }
+/**
+ * Common stylesheet for Orchestra.
+ *
+ * This partial provides common variables and mixins useful for various view and
+ * component styles.
+ */
+.disable-select {
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+  -khtml-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none; }
+
+.row.no-padding, .row.no-padding > [class*="col-"] {
+  padding-left: 0;
+  padding-right: 0; }
+
+/* Confirmation modals */
+.modal-confirm-submit {
+  padding-top: 200px; }
+
+.modal-confirm-header {
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px; }
+
+.modal-header {
+  background: #0079bf; }
+
+.modal-confirm-body {
+  overflow: auto;
+  width: 300px; }
+  .modal-confirm-body hr {
+    margin: 0px 0px 3px 0px; }
+  .modal-confirm-body .time-error-message {
+    color: red;
+    text-align: right;
+    font-size: 1.2rem;
+    padding-right: 9px; }
+  .modal-confirm-body .time-question {
+    margin-bottom: 3px; }
+  .modal-confirm-body .time-summary {
+    overflow: auto;
+    padding-bottom: 3px; }
+    .modal-confirm-body .time-summary .form-control {
+      width: 30px;
+      padding: 2px;
+      height: 23px;
+      text-align: right; }
+    .modal-confirm-body .time-summary .time-label {
+      float: left; }
+    .modal-confirm-body .time-summary .time-report {
+      float: right; }
+      .modal-confirm-body .time-summary .time-report .time-unit-group {
+        float: left;
+        width: 70px; }
+        .modal-confirm-body .time-summary .time-report .time-unit-group .time-value-float {
+          width: 30px;
+          overflow: auto;
+          margin-right: 4px;
+          float: left; }
+          .modal-confirm-body .time-summary .time-report .time-unit-group .time-value-float .time-value {
+            float: right;
+            padding-right: 2px; }
+        .modal-confirm-body .time-summary .time-report .time-unit-group .time-unit {
+          font-size: 1.2rem; }
+
+.modal-confirm-text {
+  margin-bottom: 10px;
+  font-size: 1.5rem; }
+
+/* Fixed-location informational boxes */
+.fixed-infobox-outer {
+  margin: 20px auto;
+  position: fixed;
+  top: 160px;
+  left: 0;
+  right: 0;
+  z-index: 1009; }
```

Comment:
i know ending on the same line used to be our style, but i'd vote for closing on the next line :)